### ISSUE TITLE
Password: (accessiblility) cannot access toggle button using keyboard

### ISSUE THREAD

**Issue by akshayaqburst** [primefaces/primeng#14914]
### Describe the bug

focus cannot be moved to Password's toggle button, then cannot toggle using keyboard

### Environment

Any env

### Angular version

13

### PrimeNG version

17.9

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18

### Steps to reproduce the behavior

1. go to Password toggle show case : https://primeng.org/password#togglemask
2. use tab to set focus on password field
3. hit tab several times, focus is never set on toggle password button

### Expected behavior

Toggle button should be able to be activated using keyboard
Focus should also stay on toggle button when password state is toggled.

**Comment by cetincakiroglu** [comment]
Hi,

It's made by design, the toggle icon will not receive a tab index. However, we're implementing PassThrough, enabling users to give whatever attribute they want to any element of the component. I'm closing the issue, PassThrough is in the implementation process right now, and we're planning to release PassThrough for all components within v17.

Closing the issue.
